Tests/Tests/SDImageTransformerTests.m

```diff
@@ -74,7 +74,8 @@ - (void)test04UIImageTransformRotate {
     // Fit size, may change size
     rotatedImage = [self.testImage sd_rotatedImageWithAngle:angle fitSize:YES];
     CGSize rotatedSize = CGSizeMake(ceil(300 * 1.414), ceil(300 * 1.414)); // 45º, square length * sqrt(2)
-    expect(CGSizeEqualToSize(rotatedImage.size, rotatedSize)).beTruthy();
+    expect(rotatedImage.size.width - rotatedSize.width <= 1).beTruthy();
+    expect(rotatedImage.size.height - rotatedSize.height <= 1).beTruthy();
```

Comment:
Using the `CIImage imageByApplyingTransform:` seems cause the size become `(426, 425)`, but not the `(425, 425)`....A little suck.